lock-api/src/main/metrics/metric-schema.yml

```diff
@@ -30,4 +30,10 @@ namespaces:
         type: timer
         docs: observed call duration during leader election
 
+  timestampLease:
```

Comment:
This will generate a class named the same as the other metrics in this PR. I think that's fine